content/05.conclusions.md

```diff
@@ -1,6 +1,59 @@
+In paragraphs from the Methods section, the model even understood the mathematical symbols in an equation and fixed references to them.
+We found that the most challenging section to revise was the abstract, which basically needs to describe the state-of-the-art in a field, the importance of the research questions, and briefly communicate how the proposed methods or analyses address them.
+This could be addressed either by designing better prompts (using a "few-shot learning" approach [@doi:10.1145/3386252]), which would be the fastest way, or by fine-tuning the pre-trained models using a large corpus of abstracts, which would potentially produce higher-quality results.
+We believe that the AI-based revision tool, although with some errors and minor issues, performed very well on the rest of the sections.
+As showed in the use cases, the improvements suggested by the model surpassed the mistakes that it eventually made, which can be easily fixed by a human after reviewing the suggestions.
+It is important to note, however, that our assessment of the quality of the revisions was necessarily subjective, as there could be, for example, writing styles that might not be widely shared across researchers.
+We tried to overcome this by selecting manuscripts of our own to more objectively assess the revisions, but we acknowledge that this is an important limitation of our work.
+Only time and a more extensive use of the tool will tell if an automated, AI-based revision approach for scientific literature will become a standard practice in the future.
+
+
+Our AI-based tool uses a paragraph-by-paragraph approach to revision, meaning that the model is applied to each paragraph of the manuscript *independently*, without considering the context of the paragraph.
+In addition to producing acceptable results, this divide-and-conquer approach has advantages, such as the ability to process long texts without strongly depending context length of the models.
+It was clear from our results, however, that depriving the tool from processing the context of the paragraph had some drawbacks.
+This was especially true in the Results and Methods sections, where the tool often added details that were present later in the text, or explained again an acronym that was previously defined.
+One way to address this issue would be to use chatbots, such as OpenAI's ChatGPT [@url:https://openai.com/blog/chatgpt/], that could process the entire manuscript using the same paragraph-by-paragraph approach while still considering the previously processed text.
+This would mimic the way a human reads a manuscript, inherit the benefits of processing one paragraph at a time, and potentially produce higher-quality revisions.
+
+
+In this preliminary work we focused on OpenAI's models mainly because of the simplicy in accessing the API.
```

Comment:
I think this should be combined with the above and shortened - this is really a long conclusions section.